eogrow/pipelines/import_tiff.py

```diff
@@ -0,0 +1,105 @@
+"""Implements a pipeline for importing reference data from a raster image."""
+from typing import List, Optional, Tuple
+
+import fs
+import numpy as np
+from pydantic import Field, validator
+
+from eolearn.core import CreateEOPatchTask, EONode, EOWorkflow, FeatureType, OverwritePermission, SaveTask
+from eolearn.features.feature_manipulation import SpatialResizeTask
+from eolearn.features.utils import ResizeLib, ResizeMethod, ResizeParam
+from eolearn.io import ImportFromTiffTask
+
+from ..core.pipeline import Pipeline
+from ..core.schemas import BaseSchema
+from ..utils.filter import get_patches_with_missing_features
+from ..utils.types import Feature
+from ..utils.validators import optional_field_validator, parse_dtype
+
+
+class ResizeSchema(BaseSchema):
+    """Uses resolution instead of new_size or scale_factors."""
```

Comment:
Not sure what this description tries to convey, maybe try to make it clearer.

Reply:
Old text, fixed